### ISSUE TITLE
getCounter doesn't work?

### ISSUE THREAD

**Issue by littleguga** [enjoyiacm/goodshare.js#3]
Trying to add new button with counter, but counter doesn't work. 
[Example here](http://jsfiddle.net/littleguga/6xjmhgLm/1/)(jsfiddle)
What's wrong?

**Comment by enjoyiacm** [comment]
Hello and thank you for reporting!

You mean function ``getCount()``? We have received similar issues (to mail) about wrong behavior of this function after loading DOM. Fixed this with commit ce2a7acbb004477ac9cb0cdc01ebe57cfb010985.

Please use latest version goodshare.js ``3.1.7``:
```html
<!-- Latest 3.1.x goodshare.js minify version from jsDelivr CDN -->
<script src="https://cdn.jsdelivr.net/jquery.goodshare.js/3.1.7/goodshare.min.js"></script>
```
Simple worked demo on JSFidle [here](https://jsfiddle.net/730xnkzr/#run).

**Comment by littleguga** [comment]
Sorry for useless worry. The trouble was that facebook api was loading over http, but jsfiddle over https. Thank you for your help!

```
Mixed Content: The page at 'https://fiddle.jshell.net/_display/' was loaded over HTTPS, but requested an insecure script 'http://graph.facebook.com/?id=https%3A%2F%2Ffiddle.jshell.net%2F_display%2F&callback=jQuery1110018767140875570476_1443956170730&_=1443956170731'. This request has been blocked; the content must be served over HTTPS.
```
